plugins/generator-1.18.2/forge-1.18.2/procedures/entity_makestopriding.java.ftl

```diff
@@ -0,0 +1 @@
+${input$entity}.stopRiding();
```

Comment:
Maybe add brackets for the sake of consistency?

Reply:
`entity_makeride` uses the same format as I used https://github.com/MCreator/MCreator/blob/master/plugins/generator-1.18.2/forge-1.18.2/procedures/entity_makeride.java.ftl

Reply:
I don't see the need for them